lib/rex/post/meterpreter/ui/console/command_dispatcher/extapi/clipboard.rb

```diff
@@ -308,20 +315,30 @@ def cmd_clipboard_monitor_dump(*args)
       when "-f"
         download_files = val.downcase != 'false'
       when "-p"
-        purge = val.downcase != 'false'
+        purge = true
+      when '--force'
+        force_overwrite = true
       when "-h"
         print_clipboard_monitor_dump_usage
         return true
       end
     }
+    
+    if download_path.nil? 
+      print_error("You need to specify download directory.")
+      return 1
```

Comment:
What's the `return 1` for here?  It makes sense you'd want to return and I think there's a boolean that's expected from the dispatchers. If it's necessary to return anything, I think it's supposed to be `return true`.